communication/mavlink_waypoint_handler.cpp

```diff
@@ -517,45 +518,45 @@ void Mavlink_waypoint_handler::clear_waypoint_list(Mavlink_waypoint_handler* way
 }
 
 
-void Mavlink_waypoint_handler::set_home(Mavlink_waypoint_handler* waypoint_handler, uint32_t sysid, mavlink_message_t* msg)
-{
-    mavlink_set_gps_global_origin_t packet;
-
-    if (!waypoint_handler->state_.is_armed())
-    {
-        mavlink_msg_set_gps_global_origin_decode(msg, &packet);
-
-        // Check if this message is for this system and subsystem
-        // Due to possible bug from QGroundControl, no check of target_component and compid
-        if ((uint8_t)packet.target_system == (uint8_t)sysid)
-        {
-            print_util_dbg_print("Set new home location.\r\n");
-            waypoint_handler->position_estimation_.local_position.origin.latitude = (double) packet.latitude / 10000000.0f;
-            waypoint_handler->position_estimation_.local_position.origin.longitude = (double) packet.longitude / 10000000.0f;
-            waypoint_handler->position_estimation_.local_position.origin.altitude = (float) packet.altitude / 1000.0f;
-
-            print_util_dbg_print("New Home location: (");
-            print_util_dbg_print_num(waypoint_handler->position_estimation_.local_position.origin.latitude * 10000000.0f, 10);
-            print_util_dbg_print(", ");
-            print_util_dbg_print_num(waypoint_handler->position_estimation_.local_position.origin.longitude * 10000000.0f, 10);
-            print_util_dbg_print(", ");
-            print_util_dbg_print_num(waypoint_handler->position_estimation_.local_position.origin.altitude * 1000.0f, 10);
-            print_util_dbg_print(")\r\n");
-
-
-            waypoint_handler->position_estimation_.set_new_fence_origin();
-
-            mavlink_message_t _msg;
-            mavlink_msg_gps_global_origin_pack(waypoint_handler->mavlink_stream_.sysid(),
-                                               waypoint_handler->mavlink_stream_.compid(),
-                                               &_msg,
-                                               waypoint_handler->position_estimation_.local_position.origin.latitude * 10000000.0f,
-                                               waypoint_handler->position_estimation_.local_position.origin.longitude * 10000000.0f,
-                                               waypoint_handler->position_estimation_.local_position.origin.altitude * 1000.0f);
-            waypoint_handler->mavlink_stream_.send(&_msg);
-        }
-    }
-}
+// void Mavlink_waypoint_handler::set_home(Mavlink_waypoint_handler* waypoint_handler, uint32_t sysid, mavlink_message_t* msg)
```

Comment:
Why do you comment out that part of the code ?
It was used by nicolas for swarming application where you want to share the same origin I think.
Very friendly to analyse logs afterwards !